src/Symfony/Component/Ldap/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * added `EntryManager::applyOperations`
+ * Added timeout option to `src/Symfony/Component/Ldap/Adapter/ExtLdap/ConnectionOptions.php`
```

Comment:
you should remove the `src/` part

Reply:
I've updated the path in the newest commit